DOCS/man/mpv.rst

```diff
@@ -371,6 +371,79 @@ To use this feature, you need to fill the ``menu-data`` property with menu
 definition data, and add a keybinding to run the ``context-menu`` command,
 which can be done with a user script.
 
+Image Bindings
+--------------
+
+Several key bindings are changed when viewing images:
+
+LEFT, RIGHT, UP, DOWN, h, j, k, l
+    Pan the image when it is larger than the window.
+
+Ctrl+LEFT, Ctrl+RIGHT, Ctrl+UP, Ctrl+DOWN, Ctrl+h, Ctrl+j, Ctrl+k, Ctrl+l
+    Align the image to one of the boundaries of the window when it is larger
+    than the window.
+
+= and -
+    Change the zoom.
+
++ and _
+    Change the zoom slowly.
+
+0
+    Reset the zoom.
+
+u
+    Toggle between showing the image unscaled in its original dimensions and
+    fitting it to the window.
+
+o
+    Fill the window with the image.
+
+s
+    Toggle between showing images for 5 seconds in a slideshow and keeping them
+    open forever.
+
+n
+    Go to the next playlist entry.
+
+p
+    Go to the previous playlist entry.
```

Comment:
I would rebind frame-step to next in image mode. We already have `<` and `>` to navigate. We can make them without modifier `,` `.` for images.

Reply:
, and . seem arbitrary and shoehorned just to mirror video key bindings, they don't specify a direction like < and >, what image viewer uses these key bindings? n and p (taken from sxiv) are also nice because they can be mirrored in shift+n and shift+p to navigate subplaylists. They could also be bound in the default input.conf to always use the same key bindings IMO, we bind p to cycle pause, but I don't see why you wouldn't use space for that, or why you need to hold shift for < > when there are free keys.

Reply:
But we can also just bind both, they're all unused keys for images anyway, and they don't override user bindings.

Reply:
Actually, it should be noted that these do override key bindings added by `mp.add_key_binding` without forced, because they're both builtin and the ones enabled last win. I did document how to disable these completely though.

Reply:
Also bound `HOME` and `END`.